### PR TITLE
add dynamic fetching of account data based on URL, able to click a mo…

### PR DESCRIPTION
**Pull request by deeayeen**
…nitor address to take it to address details view, and able to input an address to search it's details

### CODE REVIEW

File: src/ui/components/Table/columns.ts [hunk 1]
```diff
@@ -6,19 +6,22 @@ export type ColumnConfiguration<RecordType> = {
...
   key,
   configuration,
+  render,
 }: ColumnConfiguration<RecordType>): ColumnType<RecordType> {
   return {
     title,
     dataIndex,
     key,
     ellipsis: true,
+    render: render,
```
Comment (tjayrush) [inline src/ui/components/Table/columns.ts:24]: I don't think this is needed to be explicit. In other code, the `render` is made part of the `configuration`. I did not write the code that uses `configuration`, but I think it's a better way to proceed. Using `configuration` allows us to eliminate this exact type of interface bloat. If we want to add a new item to the interface, we simply add it to the `configuration` object and it gets used without changing the interface.

File: src/ui/hooks/useCommand.ts [hunk 2]
```diff
@@ -42,8 +42,8 @@ export function toFailedResult(error: Error): FailedResult {
 export function toFailedScrapeResult(error: Error): FailedScrapeResult {
   return {
     status: 'fail',
-    monitor: {Running: false},
-    indexer: {Running: false},
+    monitor: { Running: false },
+    indexer: { Running: false },
```
Comment (tjayrush) [inline src/ui/hooks/useCommand.ts:46]: I'd really like to figure out why these sorts of 'formatting only' changes happen whenever you edit. There must be some sort of missing setting in the prettier code. From now on, if you submit something for PR, please make sure these formatting differences aren't present.

Having said that, I want to be clear, I like your formatting much better than mine, so I wonder if there's some way to make sure when I post something, it's formatted as your code is.

I don't know which setting to change, though. Help appreciated.

File: src/ui/components/Table/columns.ts (hunk 1 above)
Comment (deeayeen) [reply src/ui/components/Table/columns.ts:24]: Fixed

File: src/ui/hooks/useCommand.ts (hunk 2 above)
Comment (deeayeen) [reply src/ui/hooks/useCommand.ts:46]: I don't know what you've got locally, all formatting settings should be in the package.json under the "prettier" key, you may have some settings overriding those

**Pull request merged**